src/ConfigurationProcessor.php

```diff
@@ -84,17 +89,27 @@ public function processInstallation(array $configuration)
             self::DEFAULT_PARALLEL_PROCESSES
         ;
 
+        $overwritingValue = false !== getenv('NO_OVERWRITE') ?
+            false :
+            self::DEFAULT_OVERWRITING_PROCESS
+        ;
+
         $processManager = new ProcessManager($updateFrequency, $processes, $this->io);
-        $shouldOverWrite = $this->io->askConfirmation('Do you want to overwrite the existing modules? (y/n)');
+        $shouldOverWrite = $this->io->askConfirmation(
+            'Do you want to overwrite the existing modules? (y/n)',
+            $overwritingValue
+        );
 
         foreach ($modules as $moduleName => $moduleVersion) {
             $package = Package::create($moduleName, $moduleVersion, $this->modulesLocation);
             $modulePath = $this->modulesLocation . $package->getName();
 
-            if ($shouldOverWrite && file_exists($modulePath)) {
-                $filesystem = new Filesystem();
-                $filesystem->remove($modulePath);
-                $this->io->write(sprintf('Module "%s" will be overwritten.', $package->getName()));
+            if ($shouldOverWrite) {
```

Comment:
the message was displayed in a weird use case: when the option was selected to 'no' (so ``$shouldOverwrite`` was false) and the module doesn't exists (false && false => true).

Not critical, but can mislead the developers...